packages/autorest.python/autorest/codegen/models/parameter_list.py

```diff
@@ -409,12 +423,14 @@ class RequestBuilderParameterList(_RequestBuilderParameterList):
 class OverloadedRequestBuilderParameterList(_RequestBuilderParameterList):
     """Parameter list for OverloadedRequestBuilder"""
 
-    def method_signature_keyword_only(self, async_mode: bool) -> List[str]:
+    def method_signature_keyword_only(
+        self, async_mode: bool, enable_signature_convert: bool = False
+    ) -> List[str]:
         """Signature for keyword only parameters"""
         if not self.keyword_only:
             return []
         return ["*,"] + [
-            parameter.method_signature(async_mode)
+            parameter.method_signature(async_mode, enable_signature_convert)
```

Comment:
Is it possible to use traceback to find out whether the serialization is in operation or request builder to avoid pass down this flag?